shared/src/main/kotlin/org/javacs/kt/classpath/Home.kt

```diff
@@ -3,6 +3,20 @@ package org.javacs.kt.classpath
 import org.javacs.kt.util.userHome
 import java.nio.file.Paths
 
-internal val gradleHome = System.getenv("GRADLE_USER_HOME")?.let { Paths.get(it) } ?: userHome.resolve(".gradle")
-// TODO: try and figure out if mavenHome is in non-default position (requires finding and parsing settings.xml)
-internal val mavenHome = userHome.resolve(".m2")
+
+internal val gradleHome =
+    System.getenv("GRADLE_USER_HOME")?.let { Paths.get(it) }
+        ?: userHome.resolve(".gradle")
+
+internal val mavenHome =
+    System.getenv("MAVEN_HOME")?.let { Paths.get(it) }
+        ?: System.getenv("M2_HOME")?.let { Paths.get(it) }
+        ?: userHome.resolve(".m2")
+
+/*
+ * TODO: try and figure out how to fetch localRepository value from actual settings.xml,
```

Comment:
Detekt doesn't like TODO-comments, which is the only reason the build is failing. (detekt was introduced not too long ago). TODO-comments are easily forgotten, and often leads to code rot after some time. I suggest you remove the comment and create an issue instead, if you don't disagree 🙂

Reply:
Removed todo-comment